src/Integration/Persistence/SolutionBinding.cs

```diff
@@ -25,20 +25,27 @@ internal class SolutionBinding : ISolutionBinding
         public const string StoreNamespace = "SonarLint.VisualStudio.Integration";
 
         public SolutionBinding(IServiceProvider serviceProvider)
-            :this(serviceProvider, null, null)
+            :this(serviceProvider, new SecretStore(StoreNamespace))
         {
         }
 
-        internal /*for testing purposes*/ SolutionBinding(IServiceProvider serviceProvider, ICredentialStore credentialStore = null, IProjectSystemHelper projectSystemHelper = null)
+        internal /*for testing purposes*/ SolutionBinding(IServiceProvider serviceProvider, ICredentialStore store)
         {
             if (serviceProvider == null)
             {
                 throw new ArgumentNullException(nameof(serviceProvider));
             }
 
+            if (store == null)
```

Comment:
Store is required , previously would replace with a real one if null to be consistent with the projectSystemHelper argument.